.github/workflows/e2e-test-elasticache.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
           SERVICE: elasticache
       - name: execute elasticache e2e tests
         run: |
+          export AWS_PROFILE=default
```

Comment:
`AWS_REGION` is set in the script (default `us-west-2`): https://github.com/aws-controllers-k8s/community/blob/main/scripts/kind-build-test.sh#L17 thus, e2e tests can be invoked by specifying aws region.
However, `daws()` function https://github.com/aws-controllers-k8s/community/blob/main/scripts/lib/aws.sh#L27 mounts the aws credentials from the host (test runner) machine to the aws cli docker container. 
As the aws credentials on the host machine can have different default aws region specified for it, the aws command would fail to find resource, if region differs.

Thus, consider setting the value for `AWS_REGION` as environment variable in github workflow.
Also, consider modifying the `daws()` method to supply the aws region value to aws cli docker container as `AWS_DEFAULT_REGION` environment variable as:

```
export AWS_DEFAULT_REGION="$AWS_REGION"
docker run --rm -v ~/.aws:/root/.aws:z $(echo $aws_cli_profile_env) -v $(pwd):/aws "$aws_cli_img" -e AWS_DEFAULT_REGION "$@"
```

Reply:
If our goal is to match the region used in `kind-build-test.sh` to the region used by `daws()`, shouldn't it be enough to add the following to `e2e-test-elasticache.yaml `? 

```
export AWS_REGION=$(aws configure get region --profile default)
```

Now that we've specified the default profile via `AWS_PROFILE` (although I'm pretty sure the default profile was being used before anyway),  and `daws()` uses the region specified in that profile, we just need to tell `kind-build-test.sh` to use that region, right?

Reply:
It would work as well, and I think its better (though it takes the flexibility of specifying the region out).
But, we shall use the profile name which is actually used on host.

Reply:
Sounds good. Just pushed a second revision with this change (along with the other requested ones).

I'll be out tomorrow, but if there are any further requested changes from you or Jay I can get to them Monday.